engines/sherlock/scalpel/scalpel.cpp

```diff
@@ -0,0 +1,764 @@
+
+	_titleOverride = "";
+	_soundOverride = "";
+	return finished;
+}
+
+/**
+ * Show the game credits
+ */
+bool ScalpelEngine::scrollCredits() {
+	// Load the images for displaying credit text
+	Common::SeekableReadStream *stream = _res->load("credits.vgs", "title.lib");
+	ImageFile creditsImages(*stream);
+	_screen->setPalette(creditsImages._palette);
+	delete stream;
+
+	// Save a copy of the screen background for use in drawing each credit frame
+	_screen->_backBuffer1.blitFrom(*_screen);
+
+	// Loop for showing the credits
+	for(int idx = 0; idx < 600 && !_events->kbHit() && !shouldQuit(); ++idx) {
+		// Copy the entire screen background before writing text
+		_screen->blitFrom(_screen->_backBuffer1);
+
+		// Write the text appropriate for the next frame
+		if (idx < 400)
+			_screen->transBlitFrom(creditsImages[0], Common::Point(10, 200 - idx), false, 0);
+		if (idx > 200)
+			_screen->transBlitFrom(creditsImages[1], Common::Point(10, 400 - idx), false, 0);
+
+		// Don't show credit text on the top and bottom ten rows of the screen
+		_screen->blitFrom(_screen->_backBuffer1, Common::Point(0, 0), Common::Rect(0, 0, SHERLOCK_SCREEN_WIDTH, 10));
+		_screen->blitFrom(_screen->_backBuffer1, Common::Point(0, SHERLOCK_SCREEN_HEIGHT - 10),
+			Common::Rect(0, SHERLOCK_SCREEN_HEIGHT - 10, SHERLOCK_SCREEN_WIDTH, SHERLOCK_SCREEN_HEIGHT));
+
+		_events->delay(100);
+	}
+
+	return true;
+}
+
+/**
+ * Show Holmes and Watson at the breakfast table, lestrade's note, and then the scrolling credits
+ */
+bool ScalpelEngine::showOfficeCutscene() {
+	_sound->playMusic("PROLOG4.MUS");
+	_titleOverride = "TITLE2.LIB";
+	_soundOverride = "TITLE.SND";
+
+	bool finished = _animation->play("COFF1", 1, 3, true, 3);
+	if (finished)
+		finished = _animation->play("COFF2", 1, 0, false, 3);
+	if (finished) {
+		showLBV("note.lbv");
+
+		if (_sound->_voices) {
+			finished = _sound->playSound("NOTE1", WAIT_KBD_OR_FINISH);
+			if (finished)
+				finished = _sound->playSound("NOTE2", WAIT_KBD_OR_FINISH);
+			if (finished)
+				finished = _sound->playSound("NOTE3", WAIT_KBD_OR_FINISH);
+			if (finished)
+				finished = _sound->playSound("NOTE4", WAIT_KBD_OR_FINISH);
+		} else
+			finished = _events->delay(19000);
+
+		_events->clearEvents();
+		finished = _events->delay(500);
+	}
+
+	if (finished)
+		finished = _animation->play("COFF3", 1, 0, true, 3);
+
+	if (finished)
+		finished = _animation->play("COFF4", 1, 0, false, 3);
+
+	if (finished)
+		finished = scrollCredits();
+
+	if (finished)
+		_screen->fadeToBlack(3);
+
+	_titleOverride = "";
+	_soundOverride = "";
+	return finished;
+}
+
+/**
+ * Load the default inventory for the game, which includes both the initial active inventory,
+ * as well as special pending inventory items which can appear automatically in the player's
+ * inventory once given required flags are set
+ */
+void ScalpelEngine::loadInventory() {
+	Inventory &inv = *_inventory;
+
+	// Initial inventory
+	inv._holdings = 2;
```

Comment:
I admit it's not the cleanest of setups, however it is done in just the one place, so it's not really the case the that the ScalpelEngine is having to "keep it consistent". In any case, _holdings is being accessed and even modified in both UserInterface and Scene. There's likely little value changing it's access here without an additional, more substantial refactoring of _holding use everywhere.